CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 Friendly reminder: We have a [bug bounty program](https://hackerone.com/cosmos).
 
+## v0.35.3
+
+April 8, 2022
+
+Special thanks to external contributors on this release:
```

Comment:
uh, marko? I think we should drop this line

Reply:
Oops, good catch, thanks. Paste leftovers.